subsys/dsp/Kconfig

```diff
@@ -32,6 +39,17 @@ config DSP_BACKEND_CUSTOM
 	  Rely on the application to provide a custom DSP backend. The implementation should be
 	  added to the 'zdsp' build target by the application or one of its modules.
 
+config DSP_BACKEND_ARC
```

Comment:
`DSP_BACKEND_ARC` sounds rather vague to me given that this backend is for the MWDT DSP library and can only be used with the MetaWare toolchain.

Something like `DSP_BACKEND_ARCMWDT` or `DSP_BACKEND_MWDTDSP` would be more adequate.